app/src/androidTest/java/net/skyscanner/backpack/compose/pageindicator/BpkPageIndicatorTest.kt

```diff
@@ -0,0 +1,83 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.pageindicator
+
+import androidx.compose.foundation.background
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.compose.BpkSnapshotTest
+import net.skyscanner.backpack.demo.R
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkPageIndicatorTest : BpkSnapshotTest() {
+
+  @Test
+  fun lessThan5pagesDefault() {
+    testPageIndicator(
+      currentIndex = 0,
+      totalIndicators = 3,
+      style = BpkPageIndicatorStyle.Default,
+    )
+  }
+
+  @Test
+  fun lessThan5pagesOverImage() {
+    testPageIndicator(
+      currentIndex = 0,
+      totalIndicators = 3,
+      style = BpkPageIndicatorStyle.OverImage,
+    )
+  }
+
+  @Test
+  fun moreThan5pagesDefault() {
+    testPageIndicator(
+      currentIndex = 4,
+      totalIndicators = 8,
+      style = BpkPageIndicatorStyle.Default,
+    )
+  }
+
+  @Test
+  fun moreThan5pagesOverImage() {
+    testPageIndicator(
+      currentIndex = 4,
+      totalIndicators = 8,
+      style = BpkPageIndicatorStyle.OverImage,
+    )
+  }
+
+  private fun testPageIndicator(
+    currentIndex: Int,
+    totalIndicators: Int,
+    style: BpkPageIndicatorStyle,
+  ) = snap {
+    BpkPageIndicator(
+      modifier = Modifier.background(color = Color.Magenta),
```

Comment:
component doesn't have a background color, making it invisible in snapshot tests.

Reply:
thanks for the suggestion. applied